### PR TITLE
DOC: Corrected file description in read_fwf()

### PR DESCRIPTION
**Pull request by isantolin**
- [ ] closes #xxxx
- [x] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by datapythonista** [comment]
Changing the name looks good, but can you please remove the link to Oracle? That document is not a spec or description of the format, it's a description of how to open fixed width files with Oracle Enterprise Data Quality, and I don't think we want to have that. It's ok to have no link if there is no good page to link to.

Thanks!

**Comment by isantolin** [comment]
@datapythonista Link removed. Can you merge?

**Comment by TomAugspurger** [comment]
Needs to pass CI first.

File: doc/source/user_guide/io.rst [hunk 1]
```diff
@@ -28,7 +28,7 @@ The pandas I/O API is a set of top level ``reader`` functions accessed like
     :delim: ;
 
     text;`CSV <https://en.wikipedia.org/wiki/Comma-separated_values>`__;:ref:`read_csv<io.read_csv_table>`;:ref:`to_csv<io.store_in_csv>`
-    text;`TXT <https://www.oracle.com/webfolder/technetwork/data-quality/edqhelp/Content/introduction/getting_started/configuring_fixed_width_text_file_formats.htm>`__;:ref:`read_fwf<io.fwf_reader>`
```
Comment (datapythonista) [inline doc/source/user_guide/io.rst]: you have like an empty list now, the backticks and underscores are not needed. After fixing that, can you also render the documentation and post a screenshot to see that it looks like expected please?

**Comment by isantolin** [comment]
@TomAugspurger , @datapythonista CI Tests passed, please merge!

**Comment by datapythonista** [comment]
@isantolin would be really appreciated if you could stop telling us when to merge. Thanks

**Pull request merged**